src/elements/Avatar.tsx

```diff
@@ -1,37 +1,102 @@
-// @ts-ignore
-import React from "react"
+import React, { ImgHTMLAttributes, SFC } from "react"
 import styled from "styled-components"
 import { space, SpaceProps } from "styled-system"
+import { color } from "../helpers/color"
+import { SerifSize } from "../Theme"
+import { Flex } from "./Flex"
+import { Serif } from "./Typography"
 
 /**
  * Spec: TODO
  */
 
-const SIZE = { xs: "45px", sm: "70px", md: "100px" }
+interface SizeProps {
+  [key: string]: {
+    diameter: string
+    typeSize: SerifSize
```

Comment:
#superMinor - but maybe this second prop would be more descriptive if it was named `serifSize`

It is much more clear already to put it into interface here!

Reply:
I think by using `serifSize` it might tie the component to the implementation a little too tightly; with `typeSize` we'd be able to swap out the font-face at a later date without worrying about updating callsites.